lib/router.js

```diff
@@ -282,11 +282,16 @@ module.exports = class Router extends express.Router {
           res.end('<script>window.close();</script>');
         })
         .catch((err) => {
-          res.status(SYSTEM_ERROR_STATUS).send(err);
+          res.cookie('unifile_error', JSON.stringify(err));
```

Comment:
I'm not very fond of passing argument via cookies. Maybe use the querystring? The URL will not be beautiful but it will better handle state and privacy browser(no cookie)

Reply:
+1
or you can render the page on the server with the error in it?

Reply:
What is the querystring please ? Are you talking about the req.query parameter ? I can not see how I can access the data in the view through this...

Reply:
Yes, that's the query. You could redirect to `/handle-errors?error=${err}` and then access the query in js. Or use hash if it's easier